Assets/Plugins/CountlySDK/CountlyUtils.cs

```diff
@@ -50,12 +50,10 @@ public string GetUniqueDeviceId()
             foreach (var item in TimeMetricModel.GetTimeMetricModel())
                 baseParams.Add(item.Key, item.Value);
 
-            if (!string.IsNullOrEmpty(_countly.OptionalParameters.CountryCode))
-                baseParams.Add("country_code", _countly.OptionalParameters.CountryCode);
-            if (!string.IsNullOrEmpty(_countly.OptionalParameters.City))
-                baseParams.Add("city", _countly.OptionalParameters.City);
-            if (_countly.OptionalParameters.Location != null)
-                baseParams.Add("location", _countly.OptionalParameters.Location);
+            if (!string.IsNullOrEmpty(_countly.Location.CountryCode))
```

Comment:
Why are you still adding these to the base params?

Reply:
Location is not added to the base params

Reply:
<img width="485" alt="Screenshot 2020-12-07 at 9 18 02 PM" src="https://user-images.githubusercontent.com/8924406/101375875-b7542f00-38d1-11eb-91ad-7ffd39632429.png">

Reply:
yes, this was still true when I mentioned it. It got fixed by your quick push